tests/unit/browser/webkit/test_history.py

```diff
@@ -347,15 +184,28 @@ def test_history_interface(qtbot, webview, hist_interface):
         webview.load(url)
 
 
+@pytest.fixture
+def cleanup_init():
+    # prevent test_init from leaking state
+    yield
+    try:
+        hist = objreg.get('web-history')
+        hist.setParent(None)
+        objreg.delete('web-history')
+        from PyQt5.QtWebKit import QWebHistoryInterface
+        QWebHistoryInterface.setDefaultInterface(None)
+    except:
+        pass
```

Comment:
This is because we can't always import `QWebHistoryInterface`. From the test below:

    try:
        from PyQt5.QtWebKit import QWebHistoryInterface
    except ImportError:
        QWebHistoryInterface = None

Also because we may not have gotten far enough to have a web-history. But I guess I can scope that more cleanly